docs/section10/10-3.md

```diff
@@ -0,0 +1,40 @@
+# Airside systems – Rule 10-3  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False   
+**Rule ID:** 10-3  
+ 
+**Rule Description:** For systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+
+**Rule Assertion:** B-RMR = expected value                                            
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. The hvac system in B-RMR serves a computer room.  
+
+**Function Call:**  
+
+1. get_zones_computer_rooms()  
+2. get_list_hvac_systems_associated_with_zone()  
+
+**Manual Check:** Check that the applicable baseline HVAC systems (hvac systems that serve comuter rooms) were modeled per the rules of G3.1-10 HVAC System baseline column which states that for systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.   
+
+**Rule Logic:**  
+**Applicability Check 1:**  
+- Call function to get dictonary of zones that include computer rooms and associated square footage in P_RMR: `zones_that_include_comp_rooms_dict_p = = get_zones_computer_rooms(P_RMR)`
+- Get list of the zones that include computer rooms: `zones_that_include_comp_rooms_list_p = zones_that_include_comp_rooms_dict_p.keys()`
+- For each zone in zones_that_include_comp_rooms_list_p: `zone_p in zones_that_include_comp_rooms_list_p:`
```

Comment:
I think for each such zone, we should check the following:
(1) If zone.maximum_humidity_setpoint_schedule is Null OR have all values equal to 1 (100% relative humidity), then pass; otherwise
(2) Find all terminals associated with each identified zone; if for each of them Terminal.heating_capacity=0 then Pass, otherwise Fail

Reply:
@mkarpman not sure why we need 1, a zone might not even have a humidity setpoint schedule, correct?
for terminal heating capacity =0, wouldnt a better check be terminal.heating_source = 'None'?

Reply:
Implemented suggestions (both MK and SG)